src/api/streams/createStream.js

```diff
@@ -9,11 +11,13 @@ export default (
   description?: string = '',
   principals?: string[] = [],
   inviteOnly?: boolean = false,
+  streamPostPolicy?: StreamPostPolicy = StreamPostPolicies.everyone,
   announce?: boolean = false,
 ): Promise<ApiResponse> =>
   apiPost(auth, 'users/me/subscriptions', {
     subscriptions: JSON.stringify([{ name, description }]),
     principals: JSON.stringify(principals),
     invite_only: inviteOnly,
+    stream_post_policy: streamPostPolicy,
```

Comment:
This shouldn't be labeled NFC -- we're changing what we send to the server.

How does this behave on older servers that don't expect this parameter? I'd expect an error.